doc/release/upcoming_changes/15769.improvement.rst

```diff
@@ -0,0 +1,15 @@
+Ability to disable madvise hugepages
+------------------------------------
+
+On Linux NumPy has previously added support for madavise
+hugepages which can improve performance for very large arrays.
+Unfortunately, on older Kernel versions this led to peformance
```

Comment:
I realise I'm late to the party since this has already been merged, but "peformance" -> "performance".